package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "zendesk",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "build": "tsc -p .",
+    "build:watch": "tsc -w -p .",
+    "start": "nodemon dist/src/index.js",
+    "test": "jest --coverage",
+    "rebuild": "rm -rf node_modules; rm package-lock.json; npm i; npm run build"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "mattermost-redux": "github:mattermost/mattermost-redux#804d4178baeb5f7745da41910bfcf2d4a393c9f0",
```

Comment:
How close is this to the latest on https://github.com/mattermost/mattermost-redux?

Reply:
fails using latest master and latest features/cloud branches.